lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -22,6 +22,10 @@ class MessageConfiguration
       SAMPLE_CONFIG_DIR                 = "#{BASE_DIR}/config-sample".freeze
       MIQ_CONFIG_DIR                    = ManageIQ::ApplianceConsole::RAILS_ROOT.join("config").freeze
 
+      def self.supported?
+        Dir.exist?(BASE_DIR)
```

Comment:
TODO not sure this is sufficient to determine if kafka is available, probably going to check for "#{BASE_DIR}/bin/kafka-server-start.sh` for the server subclass and `#{BASE_DIR}/config/client.properties` for the client.  Just worried if we dnf remove it could leave empty directories around

Reply:
`kafka-run-class.sh` is used by kafka server and kafka-console-consumer so I switched to use that